teku/src/main/java/tech/pegasys/teku/services/ServiceController.java

```diff
@@ -37,7 +39,9 @@ public ServiceController(final ServiceConfig config) {
     if (config.getConfig().isRemoteValidatorApiEnabled()) {
       services.add(new RemoteValidatorService(config));
     } else {
-      services.add(ValidatorClientService.create(config));
+      final Path slashingProtectionPath =
+          DataOptions.getValidatorsSlashingProtectionPath(config.getConfig().getDataPath());
+      services.add(ValidatorClientService.create(config, slashingProtectionPath));
```

Comment:
Calculating the slashingProtectionPath should stay inside `ValidatorClientService`.  The `ServiceController` class should know as little as possible about the details of the service and basically just create and pass the `ServiceConfig`.

Reply:
I had settled on this because DataOptions wasn't available to the validatorClient, and validatorClient wasn't available to the import/export, so I'll have to have a think about how this is all going to link together and come up with another solution i guess.

Reply:
Maybe just add a separate item to `TekuConfiguration` that has the slashing protection path.  Right now it would be calculated off of data dir but then we'd be in a good place to be able to make it a separate CLI arg in the future.